pkg/asset/installconfig/powervs/regions.go

```diff
@@ -30,7 +30,11 @@ func IsKnownRegion(region string) bool {
 }
 
 func knownZones(region string) []string {
-	return powervs.Regions[region].Zones
+	zones := []string{}
```

Comment:
Nit:
```suggestion
	zones := make([]string{}, 0, len(powervs.Regions[region].Zones))
```

Reply:
Sure, can have a strict bound. Addressed in the latest push.